en/docs/cloud/ai-workspace/llm-proxies/manage-proxy.md

```
@@ -144,14 +144,10 @@ Configure how client applications authenticate when accessing your proxy endpoin
 | Field | Description |
 |-------|-------------|
 | **Authentication type** | Select the authentication method from the dropdown. Currently, **API Key** is available. |
-| **Key name** | The name of the key that clients must provide (e.g., `X-API-Key`). |
-| **Sent in** | Choose how clients send the authentication key: **Header** (recommended) or **Query Parameter**. |
-
+| **Key name** | The name of the request header that clients must provide (e.g., `X-API-Key`). |
+| **Sent in** | Currently, `header` is the only supported option. |
```

markdownlint-cli2 (0.23.1)

[warning] Tables should be surrounded by blank lines

(MD058, blanks-around-tables)